### PR TITLE
Add check isRunning for docker

### PR DESCRIPTION
**Pull request by yJunS**

### CODE REVIEW

File: kubectl-plugin/install/installer.go [hunk 1]
```diff
@@ -24,6 +24,11 @@ You can get more details about the ks-installer from https://github.com/kubesphe
 		RunE:    opt.runE,
 	}
 
+	commander := Commander{}
+	if err := commander.execCommand("docker", "ps"); err != nil {
+		return
+	}
+
```
Comment (LinuxSuRen) [inline kubectl-plugin/install/installer.go:31]: There is no necessary to check the status of the Docker service during the command initial process.

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
You reuse the dep check of the Docker service. See also https://github.com/kubesphere-sigs/ks/pull/182/files#diff-c36cc53b299a98368f1e45af2be18bd782a94b5b9097053ddbc3d22367fd5cd0L100

It also can help to restart Docker service.

File: kubectl-plugin/install/installer.go (hunk 1 above)
Comment (yJunS) [reply kubectl-plugin/install/installer.go:31]: Yes, I have verified that it is feasible, so this PR should be closed

Comment (LinuxSuRen) [reply kubectl-plugin/install/installer.go:31]: ok

**Pull request closed without merging**